### DISCUSSION TITLE
Downloading multiple mods "Add To Cart" style

### DISCUSSION THREAD

**Discussion by MrDiamondDog** [modrinth/.github#38] [Feature requests]
Basically where you can add different mods to a download list, and when you are done looking for mods, you can download all of the mods you added to the list. The button to add a mod to a list would be on the versions of the mod. Also could make it easier to share with other people so you can all download the mods.

(concept)
![Concept](https://user-images.githubusercontent.com/84212701/224516177-3a5680e0-ad0b-4238-a3e4-77a86349c0b8.png)

**Comment by triphora** [comment]
Duplicate of #30
